app/src/main/java/com/github/fribourgsdp/radio/data/view/PlaylistSongsFragment.kt

```diff
@@ -81,7 +87,17 @@ open class PlaylistSongsFragment : MyFragment(R.layout.fragment_playlist_display
             }
         }
 
-
+        importLyricsButton = requireView().findViewById(R.id.ImportLyricsButton)
+        importLyricsButton.setOnClickListener {
+              playlist.loadLyrics().addOnSuccessListener {
+                user.addPlaylist(playlist)
+                user.save(requireContext())
+                if(playlist.savedOnline){
+                   user.onlineCopyAndSave()
+                }
+                importLyricsButton.visibility = View.INVISIBLE
+            }
+        }
```

Comment:
Would it be possible to test this? I feel like clicking a button is easy coverage we can get. You may want to be careful and try mocking the LyricsGetter to avoid real calls to MusixMatch though but you can look at what was done in the spotifyTests, w.r.t how to mock JsonParser and HttpClient and it should be fine.

Reply:
Yes, indeed. I didn't do it the way you proposed though. As load lyrics is already tested in PlaylistText, I directly mock this function.